pkg/noderesourcetopology/filter_test.go

```diff
@@ -222,14 +222,6 @@ func TestNodeResourceTopology(t *testing.T) {
 			node:       nodes[1],
 			wantStatus: nil,
 		},
-		{
-			name: "Burstable QoS, pod doesn't fit",
-			pod: makePodByResourceList(&v1.ResourceList{
-				v1.ResourceCPU:  *resource.NewQuantity(14, resource.DecimalSI),
-				nicResourceName: *resource.NewQuantity(3, resource.DecimalSI)}),
-			node:       nodes[1],
-			wantStatus: nil, // number of cpu is exceeded, but in case of burstable QoS for cpu resources we rely on fit.go
-		},
```

Comment:
Seems a bit strange to me in this test that the pod doesn't fit and yet the return nil as the status. Is that why we are removing it from here?
Also, (in this and a few other tests) for creation of burstable pod we seem to be calling [makePodByResourceList](https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/pkg/noderesourcetopology/pluginhelpers.go#L91-L104) which creates a pod such that request =limit.

Reply:
I'm removing this test because the plugin was implicitely depending on `noderesourcesfit` to perform this check. I think the plugin should be self-sufficient and check all the conditions it is meant to enforce. So the NRT filter should actually fail this test.

Reply:
Thanks for the explanation, makes sense to not have assumptions that the plugin is configured to run alongside  `noderesourcesfit`.